test/unit/single_template_test.rb

```
@@ -0,0 +1,378 @@
+      href = link["href"]
+      href.include?("jetthoughts.com") || href.include?(request_domain)
+    end
+
+    external_links.each do |link|
+      rel = link["rel"]
+
+      # External links benefit from security attributes
+      # This is a recommendation for security best practices
+      if rel
+        security_keywords = ["noopener", "noreferrer", "nofollow"]
+        has_security_attr = security_keywords.any? { |keyword| rel.include?(keyword) }
```

RuboCop (1.80.2)

[warning] Useless assignment to variable - `has_security_attr`.

(Lint/UselessAssignment)